docs-preview/package.json

```diff
@@ -66,89 +62,10 @@
-    "axios": "^0.17.1",
-    "cheerio": "^1.0.0-rc.2",
-    "child_process": "^1.0.2",
-    "command-exists": "^1.2.7",
-    "file-url": "^2.0.2",
-    "highlightjs": "^9.10.0",
-    "http-proxy-agent": "^2.1.0",
-    "https-proxy-agent": "^2.2.1",
-    "jquery": "^3.2.1",
-    "mkdirp": "^0.5.1",
-    "tmp": "0.0.33",
-    "vscode-languageclient": "^3.4.2",
-    "yauzl": "^2.9.1"
-  },
-  "runtimeDependencies": [
-    {
-      "description": "docs-previewService for .NET Core",
-      "url": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.18/netcore.zip",
-      "fallbackUrl": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.18/netcore.zip",
-      "installPath": ".markdocs",
-      "platforms": [
-        "netcore"
-      ],
-      "architectures": [
-        "netcore"
-      ],
-      "installTestPath": "./.markdocs/MarkdocsService.dll"
-    },
-    {
-      "description": "docs-previewService for Windows (x86)",
-      "url": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/win32_x86.zip",
-      "fallbackUrl": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/win32_x86.zip",
-      "installPath": ".markdocs",
-      "platforms": [
-        "win32"
-      ],
-      "architectures": [
-        "x86"
-      ],
-      "installTestPath": "./.markdocs/MarkdocsService.exe"
-    },
-    {
-      "description": "docs-previewService for Windows (x64)",
-      "url": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/win32_x64.zip",
-      "fallbackUrl": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/win32_x64.zip",
-      "installPath": ".markdocs",
-      "platforms": [
-        "win32"
-      ],
-      "architectures": [
-        "x86_64"
-      ],
-      "installTestPath": "./.markdocs/MarkdocsService.exe"
-    },
-    {
-      "description": "docs-previewService for macOS (x64)",
-      "url": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/darwin_x64.zip",
-      "fallbackUrl": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/darwin_x64.zip",
-      "installPath": ".markdocs",
-      "platforms": [
-        "darwin"
-      ],
-      "binaries": [
-        "./MarkdocsService"
-      ],
-      "installTestPath": "./.markdocs/MarkdocsService"
-    },
-    {
-      "description": "docs-previewService for Linux (x64)",
-      "url": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/linux_x64.zip",
-      "fallbackUrl": "https://github.com/qinezh/markdocs-server/releases/download/v1.0.14/linux_x64.zip",
-      "installPath": ".markdocs",
-      "platforms": [
-        "linux"
-      ],
-      "architectures": [
-        "x86_64"
-      ],
-      "binaries": [
-        "./MarkdocsService"
-      ],
-      "installTestPath": "./.markdocs/MarkdocsService"
-    }
-  ]
+    "chai": "^4.2.0",
+    "markdown-it": "^8.4.2",
+    "markdown-it-include": "^1.1.0",
+    "tslint": "^5.16.0",
```

Comment:
I think tslint should be a dev dependency?

Reply:
Good call